Example/GovUK-Onboarding.xcodeproj/project.pbxproj

```diff
@@ -3,7 +3,7 @@
 	archiveVersion = 1;
 	classes = {
 	};
-	objectVersion = 60;
+	objectVersion = 56;
```

Comment:
Same as [this](https://github.com/alphagov/govuk-mobile-ios-services/pull/3#discussion_r1579141376)